src/ui/map.js

```diff
@@ -235,7 +237,7 @@ const defaultOptions = {
  * @param {number} [options.fadeDuration=300] Controls the duration of the fade-in/fade-out animation for label collisions, in milliseconds. This setting affects all symbol layers. This setting does not affect the duration of runtime styling transitions or raster tile cross-fading.
  * @param {boolean} [options.crossSourceCollisions=true] If `true`, symbols from multiple sources can collide with each other during collision detection. If `false`, collision detection is run separately for the symbols in each source.
  * @param {string} [options.accessToken=null] If specified, map will use this token instead of the one defined in mapboxgl.accessToken.
-
+ * @param {string} [options.locale=null] A patch to apply to the default localization table. You can replace all UI strings (thereby adding support for a new translation) or just part of them (thereby patching the default translation table).
```

Comment:
This will take an `Object` value, right? Do we need to be more specific about the type of object and define (elsewhere?) what a `Locale` object is shaped like (e.g. which keys are valid)? 

In any case I think the docs entry here could be a bit more specific/descriptive, here's my attempt to flesh it out a bit.

```suggestion
 * @param {Object} [options.locale=null] A patch to apply to the default localization table for UI strings, e.g. control tooltips. The `locale` object maps namespaced UI string IDs to translated strings in the target language; see `src/ui/default_locale.js` for an example with all supported string IDs. The object may specify all UI strings (thereby adding support for a new translation) or only a subset of strings (thereby patching the default translation table).
```